pointmatcher/IO.cpp

```diff
@@ -1780,9 +1780,9 @@ template
 class PointMatcherIO<double>::PLYElement;
 
 template
-class PointMatcherIO<float>::PLYProperty;
+struct PointMatcherIO<float>::PLYProperty;
```

Comment:
What is the reason behind this change?

Reply:
I changed it because I was getting this warning in CLion :

`Class 'PLYProperty' was previously declared as a struct; this is valid, but may result in linker errors under the Microsoft C++ ABI`

Reply:
Thanks! 🙂

Now, to adress your questions :

1. I use CLion's built-in inspections, as well as clang-tidy via the built-in clangd version 11.
2. No, only under Ubuntu 18.04.